src/sentry/migrations/0450_pickle_to_json_sentry_option.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.28 on 2023-05-19 17:25
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def _backfill(apps, schema_editor):
+    Option = apps.get_model("sentry", "Option")
+
+    for obj in RangeQuerySetWrapperWithProgressBar(Option.objects.all()):
+        # load pickle, save json
+        obj.save(update_fields=["value"])
+
+
+class Migration(CheckedMigration):
+    # data migration: must be run out of band
+    is_dangerous = True
```

Comment:
`Option` has < 1k rows and `ControlOption` has 0 rows, so you could mark these not dangerous and let them run as part of the deploy if you want to save ops some hassle

Reply:
oops, well they're merged now so I guess I'll go through the hassle

is there some way I can check the row count on prod? I still need to do the following tables:
- sentry_organizationoptions
- sentry_projectoptions
- sentry_useroption

Reply:
Replied via slack